### PR TITLE
Belarusian: add rules for softening `з, с` / tests [ready to review]

### PR DESCRIPTION
**Pull request by bespsm**
soft `з, с` if previous syllable is soft
add test for the new rules

### CODE REVIEW

File: dictsource/be_rules [hunk 1]
```diff
@@ -68,6 +76,8 @@
        з (_    s
        з (L02  s
        зд (ж   z.d
+    з) з (L04  ;z
+    с) з (і    ;z
```
Comment (HaraldZealot) [inline dictsource/be_rules]: is here any reason not for `с) з (L04   ;z`?

File: dictsource/be_rules [hunk 2]
```diff
@@ -125,6 +143,8 @@
        ц       ts    // t͡s
        цц      ts:
        ц (L03  dz
+    с) ц (L04  ;ts
+    з) ц (і    ;ts
```
Comment (HaraldZealot) [inline dictsource/be_rules]: the same here

**Review by HaraldZealot — APPROVED** [review]
LGTM per se, there are only small comments in two cases, why rules are to narrow.

File: dictsource/be_rules (hunk 1 above)
Comment (HaraldZealot) [reply dictsource/be_rules]: resolved, in next commit

**Comment by HaraldZealot** [comment]
LGTM

**Comment by bespsm** [comment]
please review @jaacoppi  @kirill-jjj @alex19EP

**Pull request merged**